CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.2.0 (2021-05-27)
+
+- Added the `[skyHref]` directive for providing access control checks to hide or disable links. [#25](https://github.com/blackbaud/skyux-router/pull/25)
```

Comment:
Do we need the brackets around `skyHref`? Looking at the linked PR, I only see two instances with the brackets in the code changes but multiple instances without.